schema.sql

```
@@ -90,6 +98,7 @@ CREATE TABLE IF NOT EXISTS activity_tags (
 CREATE INDEX IF NOT EXISTS idx_activities_host      ON activities(host_id);
 CREATE INDEX IF NOT EXISTS idx_enrollments_activity ON enrollments(activity_id);
 CREATE INDEX IF NOT EXISTS idx_enrollments_user     ON enrollments(user_id);
+CREATE INDEX IF NOT EXISTS idx_certificates_enrollment ON certificates(enrollment_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)